client/scripts/main.tsx

```diff
@@ -47,8 +109,11 @@ class Order extends React.Component<any, any> {
 class Inventory extends React.Component<any, any> {
   render() {
     return (
-      <p>Inventory</p>
-    )
+      <div>
+        <h2>Inventory</h2>
+        <AddFishForm {...this.props}/>
```

Comment:
more info on the spread attributes can be found here https://facebook.github.io/react/docs/jsx-spread.html#spread-attributes